Source/WebCore/history/BackForwardItemIdentifier.h

```diff
@@ -34,14 +34,11 @@ namespace WebCore {
 
 struct BackForwardItemIdentifier {
     ProcessIdentifier processIdentifier;
-    enum ItemIdentifierType { };
+    enum class ItemIdentifierType : uint8_t { };
```

Comment:
This is suspicious.  I think BackForwardItemIdentifier should probably just be changed to use ProcessQualified.  ObjectIdentifiers are exposed through serialized_identifiers in messages.py